pegasus/helpers/email_preference_helpers.rb

```diff
@@ -47,5 +37,21 @@ def self.upsert!(email:, opt_in:, ip_address:, source:, form_kind:)
           updated_at: current_time
         }
       )
+    # Don't change opt_in to false if a preference with opt_in = true exists already.
+    # We currently only enable a user to opt out via the Pardot unsubscribe link.
+    elsif !(existing_email_preference[:opt_in] && !input_opt_in)
```

Comment:
Is this the same as 

```
elsif !existing_email_preference[:opt_in] || input_opt_in
```

and does that become easier to reason through?

Reply:
Yup!